File: src/main/java/io/github/dylmeadows/eontimer/controller/timer/CustomTimerController.java
package io.github.dylmeadows.eontimer.controller.timer;

import io.github.dylmeadows.eontimer.model.timer.CustomTimerModel;
import lombok.RequiredArgsConstructor;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;

@Component
@RequiredArgsConstructor(onConstructor_ = @Autowired)
public class CustomTimerController {

    private final CustomTimerModel model;
}

File: src/main/kotlin/io/github/dylmeadows/eontimer/controller/timer/CustomTimerController.kt
package io.github.dylmeadows.eontimer.controller.timer

import io.github.dylmeadows.eontimer.model.timer.CustomTimerModel
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Component

@Component
class CustomTimerController @Autowired constructor(
    private val model: CustomTimerModel)